client/css/style.css

```diff
@@ -17,7 +17,7 @@
 	--button-text-color-hover: #fff;
 
 	/* Color for sidebar overlay and other things that dim the viewport when something else is on top */
-	--overlay-bg-color: rgba(0, 0, 0, 0.5);
+	--overlay-bg-color: rgb(0 0 0 / 50%);
```

Comment:
I prefer rgba but 🤷‍♂️

Reply:
I've started using the `r g b / a` syntax myself since the `stylelint` project suggests doing so. I figure if a group who's taken the time to write a CSS linter thinks its the best syntax to use, I can probably just go with it.

Reply:
I try to not fight the linter if I don't have to but don't have any opinion in this case.
If any of you feel strongly say so and I'll fix up the linter to do what you want.